src/App.js

```diff
@@ -1,13 +1,89 @@
 import React from 'react';
+import './components/TodoComponents/Todo.css'
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+const todoListSeed = [
+  {
+    task: 'Learn setState()',
+    id: Date.now(),
+    completed: false
+  },
+  {
+    task: 'Style my Todo List',
+    id: Date.now() + 1,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todoList: todoListSeed,
+      inputText: '',
+      id: Date.now(),
+      completed: false
+    }
+  }
+
+  componentDidMount() {
+    const storedTodo = JSON.parse(localStorage.getItem('todo'));
```

Comment:
You are messing w/ my local storage, I don't know if it's what you're trying to save to local storage, but it is definitely saving stuff to my local storage.